src/utils/time.ts

```diff
@@ -143,6 +143,20 @@ export const formatHourAndMinute = (utcSeconds: number) => {
   return hh + ":" + mm;
 };
 
+export const formatHour = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  const hh = String(date.getHours()).padStart(2, "0");
+  return hh;
+};
+
+export const getMinutes = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  const hh = date.getHours();
+  const mm = date.getMinutes();
+  const minutes = hh * 60 + mm;
+  return minutes;
+};
```

Comment:
is the intended effect of this to get how many minutes have passed since utcSeconds?
Explain a bit more on this, please

Reply:
See comments on using `date-fns` for this in https://github.com/sparkletown/sparkle/pull/1302#discussion_r611309537

Reply:
@sunny-viktoryia And this one?

Reply:
removed the function as it is no longer used https://github.com/sparkletown/sparkle/pull/1302/commits/32efd4933865e151ac1c91f8642b9ca568bdf7c8